commit e84e3e5d27424e34b23e64d60004b9dcb99d6ea2
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-03-18T13:08:27-07:00 (7 months ago)
    
    TOOLS-2209 sdcnode http-keepAliveTimeout-default-0-v8.x.patch does not apply cleanly to node v8.11
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/nodeconfigs.json b/nodeconfigs.json
index fbd595d..bf8661f 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -83,7 +83,7 @@
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
         "version": "v8.11.0",
         "patch_files": [
-            "patches/http-keepAliveTimeout-default-0-v8.x.patch",
+            "patches/http-keepAliveTimeout-default-0-v8.11.patch",
             "patches/tls-reload-certs-v8.x.patch"
         ],
         "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
diff --git a/package.json b/package.json
index c11d34c..e9f4885 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcnode",
   "description": "builds of node for SDC",
-  "version": "1.1.1",
+  "version": "1.1.2",
   "private": true,
   "author": "Joyent",
   "engines": [
diff --git a/patches/http-keepAliveTimeout-default-0-v8.11.patch b/patches/http-keepAliveTimeout-default-0-v8.11.patch
new file mode 100644
index 0000000..9100971
--- /dev/null
+++ b/patches/http-keepAliveTimeout-default-0-v8.11.patch
@@ -0,0 +1,26 @@
+diff --git a/lib/_http_server.js b/lib/_http_server.js
+index 32c39e6160..4e8cd67aa3 100644
+--- a/lib/_http_server.js
++++ b/lib/_http_server.js
+@@ -277,7 +277,7 @@ function Server(requestListener) {
+   this.on('connection', connectionListener);
+ 
+   this.timeout = 2 * 60 * 1000;
+-  this.keepAliveTimeout = 5000;
++  this.keepAliveTimeout = 0;
+   this._pendingResponseData = 0;
+   this.maxHeadersCount = null;
+ }
+diff --git a/lib/https.js b/lib/https.js
+index 6fcd9f65ce..3ad9574338 100644
+--- a/lib/https.js
++++ b/lib/https.js
+@@ -65,7 +65,7 @@ function Server(opts, requestListener) {
+   });
+ 
+   this.timeout = 2 * 60 * 1000;
+-  this.keepAliveTimeout = 5000;
++  this.keepAliveTimeout = 0;
+ }
+ inherits(Server, tls.Server);
+ exports.Server = Server;
